### PR TITLE
Lps 92429 DS for asset-list-service

### PR DESCRIPTION
**Pull request by lanceji92674957**

### CODE REVIEW

**Comment by lanceji92674957** [comment]
ci:test:relevant

File: modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java [hunk 1]
```diff
@@ -293,18 +300,16 @@ public void updateAssetListEntryTypeSettingsProperties(
...
-		_portletResourcePermission =
-			PortletResourcePermissionFactory.getInstance(
-				AssetListEntryServiceImpl.class, "_portletResourcePermission",
-				AssetListConstants.RESOURCE_NAME);
-
-	@ServiceReference(type = CustomSQL.class)
+	@Reference(
+		target = "(model.class.name=com.liferay.asset.list.model.AssetListEntry)"
+	)
+	private volatile ModelResourcePermission<AssetListEntry>
```
Comment (Preston-Crary) [inline modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java]: This doesn't need to be volatile anymore. Can you push the change, no need to retest.

File: modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java [hunk 2]
```diff
@@ -293,18 +300,16 @@ public void updateAssetListEntryTypeSettingsProperties(
...
+		_assetListEntryModelResourcePermission;
+
+	@Reference
 	private CustomSQL _customSQL;
 
+	@Reference(target = "(resource.name=com.liferay.asset.list)")
+	private volatile PortletResourcePermission _portletResourcePermission;
```
Comment (Preston-Crary) [inline modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java]: Same here.

File: modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java (hunk 1 above)
Comment (lanceji92674957) [reply modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java]: ok, done that

**Comment by Preston-Crary** [comment]
Please see https://github.com/shuyangzhou/liferay-portal/pull/7512

**Pull request closed without merging**